quiche/src/range_buf.rs

```diff
@@ -0,0 +1,209 @@
+use std::cmp;
```

Comment:
Nit: this needs copyright+license header.

Reply:
Fixed in https://github.com/cloudflare/quiche/compare/22a538816409835b6911423db857099c501f61c4..9d9bafb0b1500e1522cd598c6c3f871006a4a451